src/SpectatorView.Unity/Assets/MarkerDetection/IMarkerVisual.cs

```diff
@@ -34,5 +34,12 @@ public interface IMarkerVisual
         /// <param name="supportedIds">Output maximum marker id</param>
         /// <returns>Returns true if the maximum marker id can be found, otherwise false.</returns>
         bool TryGetMaxSupportedMarkerId(out int markerId);
+
+        /// <summary>
+        /// Tries to set an additional scale factors for the marker visual.

```

Comment:
>Tries to set an additional scale factors for the marker visual. [](start = 12, length = 63)

I think more  description might be necessary here. How does this differ from the marker size?